### ISSUE TITLE
4.22 Android and iOS packaging warnings

### ISSUE THREAD

**Issue by Skylonxe** [TheHerobrine/ElectronicNodes#11]
**Describe the bug**
Building the plugin gives warnings on 4.22 when building for android / iOS - these use clang as compiler.

**To Reproduce**
1. Package for Android

**Expected behavior**
No warnings shown

**Screenshots**
![image](https://user-images.githubusercontent.com/8238957/69876342-9ff66500-12bf-11ea-8084-26912c2a265b.png)

**Environment (please complete the following information):**
 - Unreal Engine version: 4.22
 - Operating System: Windows

**Comment by TheHerobrine** [comment]
@Skylonxe 

From what I understand, it's just a simple warning that is not really important...

This will trigger the warning:
```cpp
void setLayerId(int32 LayerId)
{
	this->LayerId = LayerId;
}
```

This won't (replaced `LayerId` by `layerId` or anything else):
```cpp
void setLayerId(int32 layerId)
{
	this->LayerId = layerId;
}
```

cf https://forums.unrealengine.com/development-discussion/c-gameplay-programming/102044-declaration-of-inputcomponent-hides-class-member

You can disable the plugin before compilation if the warning bother you, I'll try to fix this for the next update ;)

**Comment by TheHerobrine** [comment]
This is solved for the 2.3 plugin version, now available on 4.24 ;) ! It will become available for 4.22 and 4.23 in the next weeks.
